CHANGELOG.md

```diff
@@ -6,17 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).
 
-## [unreleased] - aaaa-mm-dd
+## [unreleased] - yyyy-mm-dd
+
+## [2.0.3] - 2022-11-07
```

Comment:
I think the heal fx changelog is in the wrong version. Looking at this list of commits, the changelog should be bigger than this
https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/compare/main...develop

Reply:
Should be corrected now!